index.ejs

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title></title>
+  </head>
+  <body>
+    <% if (user) {%>
+    <h1>WELCOME BACK <%= user.username %></h1>
+    <a href="/log-out">LOG OUT</a>
+    <% } else { %>
+    <h1>Please log in</h1>
+    <% messages.forEach((msg) =>{ %>
+    <p><%= msg %></p>
+    <% }) %>
+    <form action="/log-in" method="POST">
+      <label for="username">Username</label>
+      <input name="username" placeholder="username" type="text" />
+      <label for="password">Password</label>
+      <input name="password" type="password" />
+      <button>Log In</button>
+    </form>
```

Comment:
Just an extra suggestion, it could be nice for the user to have a link to the sign up page from here

Reply:
it is done👍